lib/network/http2_network.go

```diff
@@ -156,6 +156,11 @@ func (t *HTTP2Network) AddHandler(pattern string, handler http.HandlerFunc) (rou
 		routerName = RouterNameAPI
 		prefix = pattern[len(UrlPathPrefixAPI):]
 	default:
+		// if a pattern has a suffix * , the router sets path prefix and handler
+		if strings.HasSuffix(pattern, "*") {
+			pathPrefix := strings.TrimSuffix(pattern, "*")
+			return t.router.PathPrefix(pathPrefix).Handler(handler)
+		}
```

Comment:
Yes, lt is needed for static file serving.

Reply:
Since it is static files, I'd much prefer we put it [on Github pages](https://help.github.com/articles/what-is-github-pages/) instead of bundling it into Sebak.

Reply:
@Geod24 lf we decide that, l think swag style is Unnecessary :-)